client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -154,7 +160,9 @@ export class RegistrantExtraInfoCaForm extends React.PureComponent {
 	}
 
 	getOrganizationErrorMessage() {
-		let message = ( this.state.errorMessages.organization || [] ).join( '\n' );
+		let message = this.props?.contactDetailsValidationErrors?.organization
```

Comment:
I _think_ this could safely be

```js
let message = this.props.contactDetailsValidationErrors?.organization || ( this.state.errorMessages.organization || [] ).join( '\n' );
```

Also notice that `this.props` will always exist so there's no need to put a question mark after it (the `?.` operator applies to the property just before it). `this.props?.contactDetailsValidationErrors` is equivalent to `this.props && this.props.contactDetailsValidationErrors`.